### PR TITLE
lang: better ru translation

### PR DESCRIPTION
**Pull request by bookmist**
Now it's ready for review. Big thanks to Kane

### CODE REVIEW

File: lang/system/ru.xml [hunk 1]
```diff
@@ -532,15 +581,205 @@
...
+			<clear_list>Очистить список</clear_list>
+			<no_modules>Модули не найдены.
+Пожалуйста, загрузите и установите последнюю прошивку PS Vita.</no_modules>.
+			<refresh_list>Обновить список</refresh_list>
+            <automatic>Автоматический</automatic>
+            <automatic_description>Выберите Автоматический чтобы использовать встроенный список модулей.</automatic_description>
+            <auto_manual>Авто и ручной</auto_manual>
+            <auto_manual_description>Выберите этот режим для загрузки автоматических модулей и выбранных модулей из списка ниже.</auto_manual_description>
+            <manual>Ручной</manual>
+            <manual_description>Выберите Ручной режим для загрузки только выбранных модулей из списка ниже.</manual_description>
```
Comment (Macdu) [inline lang/system/ru.xml:598]: Indentation issue here

File: lang/system/ru.xml [hunk 2]
```diff
@@ -532,15 +581,205 @@
...
+			<spirv_shader_description>Передавать сгенерированные шейдеры Spir-V непосредственно в драйвер.
+Обратите внимание, что некоторые полезные расширения будут отключены,
+и не все графические процессоры совместимы с этим режимом.</spirv_shader_description>
+			<clean_shaders>Очистить кэш и лог шейдеров</clean_shaders>
+			<fps_hack>FPS хак</fps_hack>
+			<fps_hack_description>Игровой хак, который позволяет некоторым играм, работающим в 30 FPS, работать в 60 FPS на эмуляторе.
+Обратите внимание, что это хак, и он будет работать только в некоторых играх.
+На другие игры он может не повлиять или заставить их работать в два раза быстрее.</fps_hack_description>
+		</gpu>	
+<audio name="Звук">
```
Comment (Macdu) [inline lang/system/ru.xml:656]: Indentation missing here

File: lang/system/ru.xml (hunk 1 above)
Comment (bookmist) [reply lang/system/ru.xml]: Fixed.

File: lang/system/ru.xml (hunk 2 above)
Comment (bookmist) [reply lang/system/ru.xml]: Fixed.

File: lang/system/ru.xml [hunk 3]
```diff
@@ -229,10 +234,20 @@
 	<compat_db>
 		<error>Ошибка</error>
 		<information>Информация</information>
+		<get_failed>Не удалось получить текущую базу данных совместимости, проверьте брандмауэр/доступ к Интернету, повторите попытку позже.</get_failed>
+		<download_failed>Не удалось загрузить базу данных совместимости приложений, обновленную на: {}, повторите попытку позже.</download_failed>
+		<load_failed>Не удалось открыть базу данных совместимости приложений, обновленную на: {}.</load_failed>
+		<new_app_listed>База данных совместимости была успешно обновлена с {} до {}.
+В списке появилось {} новых приложений, в результате чего общее количество приложений достигло {}!</new_app_listed>
+		<app_listed>База данных совместимости была успешно обновлена с {} до {}.
+В списке {} приложений! </app_listed>
```
Comment (nishinji) [inline lang/system/ru.xml:243]:
```suggestion
В списке {} приложений!</app_listed>
```

File: lang/system/ru.xml [hunk 4]
```diff
@@ -532,15 +581,205 @@
...
 
 	<settings_dialog name="Настройки">
+		<core name="Ядро">
+			<modules_mode>Режим загрузки модулей</modules_mode>
+			<modules_list>Список модулей</modules_list>
+			<select_modules>Выберите нужные модули.</select_modules>
+			<search_modules>Поиск модулей</search_modules>
+			<clear_list>Очистить список</clear_list>
+			<no_modules>Модули не найдены.
+Пожалуйста, загрузите и установите последнюю прошивку PS Vita.</no_modules>.
```
Comment (nishinji) [inline lang/system/ru.xml:591]:
```suggestion
Пожалуйста, загрузите и установите последнюю прошивку PS Vita.</no_modules>
```

Comment (nishinji) [reply lang/system/ru.xml:591]: Some of the same mistakes.

File: lang/system/ru.xml (hunk 3 above)
Comment (bookmist) [reply lang/system/ru.xml]: Fixed.

File: lang/system/ru.xml (hunk 4 above)
Comment (bookmist) [reply lang/system/ru.xml]: Fixed all such mistakes. Thank you.

**Review by Macdu — APPROVED** [review]

**Pull request merged**